chemked/chemked.py

```diff
@@ -95,6 +95,102 @@ def validate_yaml(self, properties):
 
             raise ValueError(v.errors)
 
+    def get_dataframe(self, output_columns=None):
+        """Get a Pandas DataFrame of the datapoints in this instance.
+
+        Arguments:
+            output_columns (list, optional): List of strings specifying the columns to include
+                in the output DataFrame. The default is ``None``, which outputs all of the
+                columns. Options include (not case sensitive):
+                    * Temperature
+                    * Pressure
+                    * Ignition Delay
+                    * Composition
+                    * Reference
+                    * Apparatus
+                    * Experiment Type
+                    * File Author
+                    * File Version
+                    * ChemKED Version
+                In addition, specific fields from the Reference and Apparatus attributes can
+                be included by specifying the name after a colon. These options are:
+                    * Reference:Volume
+                    * Reference:Journal
+                    * Reference:DOI
+                    * Reference:Authors
+                    * Reference:Detail
+                    * Reference:Year
+                    * Reference:Pages
+                    * Apparatus:Kind
+                    * Apparatus:Facility
+                    * Apparatus:Institution
+                Only the first author is printed when ``Reference`` or ``Reference:Authors`` is
+                selected because the whole author list may be quite long.
+
+        Returns:
+            DataFrame: Contains the information regarding each point in the ``datapoints`` attribute
+        """
+        import pandas as pd
```

Comment:
I meant to respond to this... I actually find it easier to read with the abbreviation, perhaps because they're familiar to me, but I think also because the abbreviations take up fewer characters to distract from what the code is doing. Anyways, you merged it, so I assume its OK to leave it.